### PR TITLE
testing/mediainfo: new aport

### PR DESCRIPTION
**Pull request by albap**
Note - I don't want to be contributor or maintainer so it'd be nice if someone who likes mediainfo could take care of.
Thanks.

### CODE REVIEW

File: testing/mediainfo/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+pkgrel="0"
+pkgdesc="Unified display of the most relevant technical and tag data for video and audio files."
+url="https://mediaarea.net/"
+arch="all"
+license="Simplified BSD"
+depends=""
+makedepends=""
+install=""
+subpackages=""
+source="https://mediaarea.net/download/binary/mediainfo/$pkgver/MediaInfo_CLI_$pkgver_GNU_FromSource.tar.xz"
```
Comment (jirutka) [inline testing/mediainfo/APKBUILD]: There’s no such variable `$pkgver_GNU_FromSource`… You must wrap it in braces in this case: `MediaInfo_CLI_${pkgver}_GNU_FromSource`.

BTW It’s really not needed nor desirable to write checksums by hand. That’d be really annoying and unnecessary, so we have a tool for that – `abuild checksum`. Have you read [guide](https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#APKBUILD_variables.2Ffunctions) on the Alpine wiki?

**Comment by albap** [comment]
@coreyjonoliver wants to maintain this?

BTW I looked at http://git.alpinelinux.org/cgit/aports/tree/testing/libmediainfo/APKBUILD for you and there are mistakes.
Website is wrong, license is wrong, version is old.

Since you maintain libmediainfo I think you could want to maintain mediainfo too?

**Comment by coreyjonoliver** [comment]
@albap I'd be willing to also maintain mediainfo.

Also, thanks for pointing out the mistakes in libmediainfo. When I have some time, I'll submit a patch to address them.

**Comment by albap** [comment]
Added you as the maintainer - nice.

**Pull request closed without merging**